### PR TITLE
Notifications and UI improvement

### PR DESCRIPTION
**Pull request by Aryan-Rusia-2**
This PR adds the following functionalities:

- Users can now receive notifications in the foreground as well.
- Clicking on notifications navigates to the desired notification detail page.
- Attendees will now also receive milestone notifications.
- Added an auto-refresh feature to the notification display page.
- Fixed the default event poster color in dark mode.

### CODE REVIEW

File: code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/AttendeeMainActivity.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public class AttendeeMainActivity extends AppCompatActivity {
     private QRCodeScanner scanner;
     private String attUUID;
     private String profileName;
-
+    NotificationManager notificationManager;
```
Comment (dpardell) [inline code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/AttendeeMainActivity.java:47]: This should probably be private

**Review by adrienz1 — APPROVED** [review]

File: code/EventLinkQR/functions/index.js [hunk 2]
```diff
@@ -109,19 +141,48 @@ exports.sendNotificationToEventAttendees = functions.firestore
...
+                        if (resp.success) {
+                            console.log(`Message sent to token ${uniqueTokens[idx]}`);
+                        } else {
+                            console.error(`Failed to send message to token ${uniqueTokens[idx]}`, resp.error);
+                        }
+                    });
+                } catch (error) {
+                    console.error('Error sending multicast messages:', error);
+                }
+                // const response = await admin.messaging().sendEachForMulticast(messages);
```
Comment (dpardell) [inline code/EventLinkQR/functions/index.js:181]: Is this supposed to be commented out?

Comment (Aryan-Rusia-2) [reply code/EventLinkQR/functions/index.js]: Sorry, my bad. I have removed those lines now.

File: code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/AttendeeMainActivity.java (hunk 1 above)
Comment (Aryan-Rusia-2) [reply code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/AttendeeMainActivity.java]: Done 👍

**Pull request merged**